README.md

```diff
@@ -35,6 +35,12 @@ Missing features needed to write the oomph compiler in oomph:
 - `==` operator for optionals
 
 Other missing features:
+- exporting classes
+- exporting unions
```

Comment:
These are needed for writing the compiler in oomph. We also need `==` for pointer types and being able to run subprocesses.